Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -2258,3 +2254,26 @@ std::string vtkMRMLMarkupsNode::GetPropertiesLabelText()
 {
   return this->PropertiesLabelText;
 }
+
+//---------------------------------------------------------------------------
+void vtkMRMLMarkupsNode::SetCurveGenerator(vtkCurveGenerator* curveGenerator)
+{
+  if (this->CurveGenerator)
+    {
+    this->CurveGenerator->RemoveObserver(this->MRMLCallbackCommand);
+    }
+
+  if (curveGenerator == nullptr)
+    {
+    this->CurveGenerator = vtkSmartPointer<vtkCurveGenerator>::New();
```

Comment:
Is this case really needed ?

Reply:
Sorry for the noise. I didn't mean to commit those changes.